docs/standard/basetypes/conversio/conversiontables.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Type Conversion Tables
```

Comment:
Why is this in a "conversio" folder - is that a typo for "conversion"?  Also there's only one file in the folder, so we probably don't need the folder.